designs/developer/operator-pipeline-45/pipelines.md

```diff
@@ -0,0 +1,244 @@
+![PLR Delete Confirmation](http://openshift.github.io/openshift-origin-design/designs/developer/operator-pipeline-42/img/PL-Details-PLR-DeleteConfirmation.png)
+
+# Task Status Component
+## Overview
+- The task status component is  a 100% stacked bar chart (Implementation note: use PF4/Victory to remain aligned with OpenShift console strategy).
+- Last status represents the tasks of the last pipeline run.
+- Each segment of the chart represents the group of tasks which are the same status.
+- Task segments are in the following order:
+    - Succeeded
+    - Failed
+    - Cancelled
+    - Skipped
+    - Running
+    - Pending
+- Hovering over the widget will show a tooltip indicating the number of tasks in each of the task statuses
+- The height of the task status component should remain constant and similar in height to associated text.
+- Clicking on any part of the task status widget will navigate to the Log Tab of the pipeline run details view.
+- Use a 3 px separator between segments
+
+# Pipeline Visualization
+## Overview
+This visualizes all tasks associated with the pipeline.
+
+![PL Visualization](http://openshift.github.io/openshift-origin-design/designs/developer/operator-pipeline-42/img/PLVisualization.png)
+
+
+# Pipeline Run Visualization
+- This visualizes the status of the tasks and associated steps of the pipeline run.
+- Tasks can be asynchronous and are represented by bubbles in the visualization
+- Steps are synchronous and are represented as line segments inside the task bubbles
+- In 4.2, only the tasks which have started will be able to be shown in the Pipeline Run visualization.
+
+![PLR Visualization](img/PLRVisualization.png)
+
+## Pipeline experiences when user has view only access of a project#
+
+_user cannot add, edit and delete any annotations, labels, parameter, and resources & user cannot start, stop, restart, edit and delete any pipeline tasks of a project when user has view access to that project_
+
+### Pipelines List View ###
+
+When a user has view only access for a project then the kebab and all associated actions will be disabled. So that user cannot start or stop or delete any of the pipelines and pipeline Runs
+
+![Pipelines List View](http://openshift.github.io/openshift-origin-design/designs/developer/operator-pipeline-42/img/RBAC-PL-ListView.png)
+
+### Pipelines Details Page / Overview ###
+
+When a user has view only access for a project then the `action` button will be hidden. So that user cannot start, restart, modify and delete any of the pipeline runs.
+
+![Pipelines Overview tab](http://openshift.github.io/openshift-origin-design/designs/developer/operator-pipeline-42/img/RBAC-PL-Details-Overview-Actions.png)
+
+### Pipelines Details Page / YAML ###
+
+When a user has view only access for a project then the `action` and `save` buttons will be hidden to prevent YAML edit.
+
+![Pipelines YAML tab](img/RBAC-PL-Details-YAML.png)
```

Comment:
if you keep the relative path here, you'll have to add the image in the designs/developer/operator-pipeline-45/img directory, or you can point back to the 4.4 image.

Reply:
OK. I will update this one to an absolute path.